app/main.py

```diff
@@ -1,6 +1,19 @@
-from typing import Callable
+from typing import Callable, Any
```

Comment:
There is a syntax error on this line. The import statement should separate multiple imports with a comma, not a slash. It should be `from typing import Callable, Any`.